content/getting-started/signup.md

```diff
@@ -43,9 +43,9 @@ If you revoke access to previously shared repositories, you will still see all y
 
 ### Disconnecting Codemagic GitHub app integration
 
-The Codemagic CI/CD GitHub App integration can be disabled by clicking **Manage integration > Disconnect** in **User settings > Integrations > GitHub App** for your personal account or in **Team settings > Team Integrations > GitHub App** to disconnect it from the team. This annuls all settings configured for the Codemagic app in GitHub but won't delete your buid history. 
+The GitHub app integration can be removed when you **uninstall** the Codemagic CI/CD app in GitHub. You can still log in to Codemagic with the GitHub App and see your build history if Codemagic CI/CD remains an **authorized GitHub app** in GitHub. 
```

Comment:
Build history can be seen even if one unauthorizes GitHub App.

Reply:
The Disconnect button will remain on the dialogue. Clicking it will delete the GitHub access token which means we no longer interact with GitHub, what changed is that we no longer revoke the authorization or installation of the GHA.